docs/user/science.rst

```diff
@@ -0,0 +1,65 @@
+Read the Docs for Science
+=========================
+
+Documentation and technical writing are broad fields that include many science activities - building publications, academic writing, educational resources and interactive data science and data journalism.
+
+Let's explore the overlap between features needed for software documentation and for instance academic writing. Here's a brief overview of some of the features that people in science and academic writing love about the many tools used on Read the Docs:
+
+🪄 Easy to use
+  You don't have to be a programmer. Use familiar Markdown-derived syntax and write your formulas with LaTeX.
+🔣 Formulas and proofs
+  Use symbols familiar from math and physics, build advanced proofs.
+🧮 Results directly imported
+  Graphs, tables etc. are computed when the latest version of your website is built. Your results can be directly imported in their latest versions.
```

Comment:
What is directly imported? I feel like there's better wording here, but I don't know exactly what it's trying to convey.

Perhaps:

```suggestion
  Graphs, tables etc. are computed when the latest version of your website is built. All code examples on your website are validated each time you build.
```